itou/templates/siaes/job_description_list.html

```diff
@@ -1,204 +1,143 @@
 {% extends "layout/content.html" %}
+{% load static %}
 {% load bootstrap4 %}
 
 {% block title %}Vos fiches de postes {{ block.super }}{% endblock %}
 
-{% block content_container_css_classes %}pt-0{% endblock %}
-
-{% block breadcrumbs %}
-    {% include "layout/breadcrumbs_from_dashboard.html" %}
+{% block pre_content %}
+    <section class="s-title-01 p-0">
+        <div class="s-title-01__container container">
+            <div class="s-title-01__row row">
+                <div class="s-title-01__col col-12">
+                    <h1 class="s-title-01__title h1">
+                        <strong>Métiers et recrutements</strong>
+                    </h1>
+                    <h2 class="h2">{{ siae.kind }} {{ siae.display_name }}</h2>
+                </div>
+            </div>
+        </div>
+    </section>
 {% endblock %}
 
+{% block content_container_css_classes %}s-section pt-0 mt-0{% endblock %}
 {% block content %}
-
-    <h1 class="mb-0">{{ siae.kind }} {{ siae.display_name }}</h1>
-    <h2 class="text-muted">Gérer les métiers et recrutements</h2>
-
-    <div class="shadow rounded p-4">
-        <div class="row">
-            <div class="col">
-                <h6>
-                    Le{{ job_pager.paginator.count|pluralize }} métier{{ job_pager.paginator.count|pluralize }} exercé{{ job_pager.paginator.count|pluralize }} dans votre structure
-                </h6>
-            </div>
-            <div class="col text-right">
-                <a class="btn btn-primary btn-sm" href="{% url "siaes_views:edit_job_description" %}?page={{ page|default:1 }}" aria-label="Créer une nouvelle fiche de poste">Créer une nouvelle fiche de poste</a>
+    <div class="c-box">
+        <div class="d-flex flex-column flex-md-row align-items-md-center justify-content-md-between">
+            <h3 class="h4 mb-0">
+                Métier{{ job_pager.paginator.count|pluralize }} exercé{{ job_pager.paginator.count|pluralize }}
+            </h3>
+            <div class="flex-column flex-md-row btn-group btn-group-sm btn-group-action" role="group" aria-label="Actions sur les candidatures">
+                {% if job_pager %}
+                    <form method="post" action="?page={{ page|default:1 }}" id="block_job_applications_form">
```

Comment:
Sur le moment j'ai un doute pour les paramètres GET mais pour moi pas besoin de forger l'URL actuel, il suffit de virer `action`.
```suggestion
                    <form method="post" id="block_job_applications_form">
```